src/Compilers/CSharp/Test/Syntax/Parsing/RawInterpolatedStringLiteralCompilingTests.cs

```diff
@@ -125,6 +125,17 @@ public void TestInConstantFieldInitializer3()
 }").VerifyDiagnostics();
     }
 
+    [Fact]
+    public void TestInConstantFieldInitializer4()
+    {
+        CreateCompilation(
+@"class C
+{
+    const string x = ""bar"";
+    const string s = $""""""{bar}"""""";
+}").VerifyDiagnostics();
+    }
```

Comment:
Thanks for adding this.
Can we do all the remaining tests in one PR? It will be easier to button things up this way, I think.

Reply:
sure :)